CACHE_INTEGRATION_GUIDE.md

```
@@ -0,0 +1,330 @@
+# 🧠 Cache Integration Guide for psm_lib.py & survival_lib.py
+
+**Status:** ✅ Ready to integrate  
+**Expected Speedup:** 30-60 seconds per repeated analysis  
+**Effort:** 5-10 minutes per module  
+
+---
+
+## 🎯 Overview
+
+Two cache integration helpers are ready to use:
+- `utils/psm_cache_integration.py` - For propensity score matching
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)